src/libraries/System.Private.CoreLib/src/System/ObjectDisposedException.cs

```diff
@@ -44,6 +46,15 @@ protected ObjectDisposedException(SerializationInfo info, StreamingContext conte
             _objectName = info.GetString("ObjectName");
         }
 
+        [StackTraceHidden]
+        [DoesNotReturn]
+        public static void Throw(object? instance) =>
+            Throw(instance?.GetType());
+
+        [DoesNotReturn]
+        public static void Throw(Type? type) =>
```

Comment:
Should this be nullable?

Reply:
I suggested it was nullable as the existing constructors allow null, and it can't be relied upon for null checking as the consuming app might not have nullable enabled. Otherwise they'd get a `NullReferenceException` if the type/object passed to the new methods was null (without adding a null check guard to the new methods, which seems to go against the null-permissive existing constructos).